UPM/Documentation~/README.md

```diff
@@ -1,4 +1,3 @@
 # Microsoft Mixed Reality Toolkit
 
-[Getting Started](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/mrtk-getting-started)
-[MRTK Documentation on GitHub](https://microsoft.github.io/MixedRealityToolkit-Unity/README.html)
```

Comment:
I think there's value in keeping a link to https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/mrtk-getting-started. It's a different set of information, and slightly more tailored to "getting started" than the mrtk-unity page

Reply:
restored